src/constants.ts

```diff
@@ -31,67 +33,101 @@ export const kParentCollectionAttributes = [
   {
     name: "longitude",
     type: "numeric"
-
   },
   {
     name: "location",
     type: "categorical"
-
   }
-  // NOTE: If data are to be historical, add year attribute
 ];
 
 export const kChildCollectionAttributes = [
   {
     name: "date",
     title: "Date",
     type: "date",
-    hasToken: true
+    hasToken: true,
+    precision: "day",
+    description: "Date"
   },
   {
-    name: "dayLength",
-    title: "Day Length",
+    name: "Day length",
+    title: "Day length",
```

Comment:
Spaces in attribute names can be tricky to use in formulas. Try it out in v2 and see if you get the behavior you want. (I say in v2 because where it matters is the formula editor, and v2 has a real formula editor which v3 doesn't have yet.)

Reply:
Oh thanks - I did try it in v2 and it worked in something basic like `round`.  At the moment the only fields being used in formulas are the (now hidden) date fields `rawSunrise` and `rawSunset`.